### DISCUSSION TITLE
Show off your APP!

### DISCUSSION THREAD

**Discussion by phusting** [apache/cordova#362] [Show and tell]
Let's see your apps that you've created in Cordova along with a little history.  I'll go first.

I started the Equestrian App back in 2018 as I had a need for having all my horse healthcare records digitally.  I shopped around but didn't like anything out there, they looked like they were written in 1998.  So I wrote it for myself.  After a bunch of my horse friends asked to use it, I rewrote it and it's used around the world in the care of horses.

website: [](https://equestrianapp.com )

Seriously, Cordova made my life as a developer so that I could just write it once and publish on 3 platforms.  

![apple-app-store-template](https://user-images.githubusercontent.com/12736097/206869576-55891135-d237-424e-8233-63e1b097b15e.png)

**Comment by j-crosson** [comment]
Interesting App.  I'm hesitant to showcase my apps because they all belong to  clients--the oldest has been in the stores since 2012--but I do have a plugin that is all mine: cordova-plugin-splitview.  

This plugin frames webViews in native split views.  Trying to fake native framing in iOS is hard and sometimes looks really bad.  Back in iOS 4 or 5 when Apple did a cosmetic update, my apps had to be updated to mimic the new back buttons and whatnot.  Native framing would have eliminated the need to update.  Framing hasn't been a big issue for me on Android, so the plugin is iOS only. 

Anyway, here is an example of a split view with a plugin-provided JavaScript-controlled native list control as the primary view. The secondary view is a Cordova webView.
  
![collectionViewFull](https://user-images.githubusercontent.com/67383519/218606025-d0b7f0c5-a24a-4fd9-a147-912e5ea344e3.png)

**Comment by breautek** [comment]
My apps are for [TotalPave](https://totalpave.com/)

We have 2 main apps both are used to collect road condition data, one using the standard PCI (Pavement Condition Index) and another collects the IRI (International Roughness Index). Available on both iOS and Android.

We deal with some extensively large data sets sometimes, so I think we really push the boundaries the webview capabilities. In some cases, we have to build specialized cordova plugins so that we can move our business logic to the native side and avoid transferring data to the webview, when it's not truly needed (because it's only used in some native views like Google Maps).

Both apps are offline capable, which means assuming that you have logged in with a data connection before, you can use the app to collect data even without a data connection.

Here is some screenshots of the PCI app:

<div>
<img src="https://user-images.githubusercontent.com/11200662/219415852-06f43e56-7353-4349-b1e7-8c3e260d9512.png" height="500" style="display:inline-block"/>
<img src="https://user-images.githubusercontent.com/11200662/219420687-afd0d9c5-8b77-47c5-957a-f00c54007bbf.png" height=500" style="display: inline-block" />
<img src="https://user-images.githubusercontent.com/11200662/219420689-f26c783b-c5bc-431a-a9ad-9751c0476d28.png" height="500" style="display: inline-block" />
<img src="https://user-images.githubusercontent.com/11200662/219420691-5d8396b3-fad1-4ff0-aa7d-2e03665ea00b.png" height="500" style="display: inline-block" />
</div>


And here is some screenshots of the IRI App (An upcoming version of it anyway):


<div>
<img src="https://user-images.githubusercontent.com/11200662/219438938-00606d34-0837-44b4-89eb-7ba094300e43.png" height="500" style="display:inline-block"/>
<img src="https://user-images.githubusercontent.com/11200662/219438941-3408237d-7d34-41da-92c4-31a111a66854.png" height="500" style="display:inline-block"/>
<img src="https://user-images.githubusercontent.com/11200662/219438943-2c694ee0-005c-4290-950c-9ce5f5e77271.png" height="500" style="display:inline-block"/>
<img src="https://user-images.githubusercontent.com/11200662/219438945-e86c55f5-e410-46f8-b3f9-4eb7afdfbc83.png" height="500" style="display:inline-block"/>
</div>

The PCI app is at its core, a simple form input app. It's used by workers on the field to do visual inspections of the roads and record problems.

The IRI app is harder to show with pictures. It's intended to mount the app in the car and you start recording data. While recording data, the worker drives around on their roads and the app will record the vibration data which is then used to calculate the IRI value from.

With both of these apps, gives municipalities a tool set to learn the behaviour and collect the data necessary to make decisions on how to maintain their roads. The collected data is then displayed on a web portal where these decision makers can review the data, create analytics and workplans, etc.

**Comment by yoldar** [comment]
Hi everyone! We started to use Apache Cordova from 2018. The first project was for construction company. We used Camunda BPM as backend for quality control. The name of app is Opera Build. Then in 2020 we started new project called re:Kassa 3.0 free online cash register for individual entrepreneurs and limited liability partnerships in Kazakhstan. A convenient and simple application that allows you to conduct transactions directly on your smartphone, generate receipts and transfer them for fiscalization. We use a lot of plugins in app. Some of them are self-made. [https://rekassa.kz/](https://rekassa.kz/)

https://github.com/apache/cordova/assets/1597578/07c9d5ea-a5be-4334-9d8d-3d427845b7fb
